extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -59,6 +59,23 @@ public static function onArticleSaveComplete( WikiPage $wikiPage, User $user, $t
 	}
 
 	/**
+	 * @param $categoryInserts
+	 * @param $categoryDeletes
+	 * @param Title $categoryTitle
+	 * @return bool
+	 */
+	static public function onAfterCategoriesUpdate( $categoryInserts, $categoryDeletes, $categoryTitle ) {
```

Comment:
Last parameter is probably not used. It's even more confusing because we override it's value when using as local variable/